dashboard/lib/pd/foorm/survey_reporter.rb

```diff
@@ -94,17 +135,15 @@ def self.parse_and_summarize_forms(ws_submissions, form_submissions, forms)
       [parsed_forms, summarized_answers]
     end
 
-    # TODO: once we store facilitator data
-    # def self.get_rollup_for_facilitator(workshop_id, facilitator_id)
-    # end
-
     # Gets the raw data needed for summarizing workshop survey results.
     # @param workshop id, the workshop to get data from
-    # @return array of [WorkshopSurveyFoormSubmissions, FoormSubmissions and FoormForms]
+    # @return array of [WorkshopSurveyFoormSubmissions, FoormSubmissions, FoormForms]
     #   for the given workshop id.
-    def self.get_raw_data_for_workshop(workshop_id)
+    def self.get_raw_data_for_workshop(workshop_id, facilitator_id=nil)
       ws_submissions = Pd::WorkshopSurveyFoormSubmission.where(pd_workshop_id: workshop_id)
-
+      if facilitator_id
+        ws_submissions = ws_submissions.where(facilitator_id: facilitator_id).or(ws_submissions.where(facilitator_id: nil))
```

Comment:
Should `pd_workshop_survey_foorm_submissions` have an index on `facilitator_id` to facilitate :smirk: this query?